### PR TITLE
Add ability to calculate stem images from electron counted data

### PR DESCRIPTION
**Pull request by psavery**
This is a work in progress currently.

Fixes: #40

### CODE REVIEW

**Comment by psavery** [comment]
This might potentially be working. I am able to create some dark and bright field image files from our electron counted data. But they look like static...

dark
bright

File: python/stempy/io/__init__.py [hunk 1]
```diff
@@ -37,6 +37,14 @@ def read(self):
...
     return Reader(path, version)
 
+def expand_electron_data(frames, numPixels):
+    ret = np.zeros((len(frames) * numPixels,), dtype=np.uint16)
+    for i, frame in enumerate(frames):
+        for pos in frame:
+            ret[i * numPixels + pos] = 1
+
+    return ret
+
```
Comment (cjh1) [inline python/stempy/io/__init__.py]: Shouldn't we do this expansion in C++, given the looping?

Comment (psavery) [reply python/stempy/io/__init__.py]: We can move it to C++, although I don't think this function takes up too much time. ~0.19 seconds in my timing (the STEM calculation is taking around 50 seconds).

Comment (cjh1) [reply python/stempy/io/__init__.py]: ok, that is fine, leave for the moment

Comment (cjh1) [reply python/stempy/io/__init__.py]: 50 seconds on the electron counted data?

**Comment by psavery** [comment]
The latest commit results in a very dramatic speed increase, from
~50 - 60 seconds on my computer to 0.5 seconds.

Timings seem to indicate that the vast majority of time
was spent in between the pybind11 function call and the start
of the C++ function. Whatever was happening in between (perhaps
a data copy or conversion) was taking a long time.

It is now much faster.

**Pull request merged**